### PR TITLE
Webinars & AMA Sessions

### LINKED ISSUE #326: Create webinar page
### Story

_No response_

### Description

I would like to create a page for our webinars that will have threemain parts:
1) top part will be promotion of the upcoming webinar and a Hubspot registration form,
2) promotion of our monthly office hours and a Hubspot registration form, 
3) an archive of previous webinars that are links to the recording on our Youtube channel.

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

- **Navigation**: We now move "Blog" alongside our new "Webinars" page underneath "Resources"
- **Webinars**: Our new "Webinars" page advertises the next upcoming Webinar, and next upcoming "Ask Me Anything" session.
- **Webinar/AMA Page**: Each session has an associated page which displays the event description, title, subtitle, hosts & a HubSpot registration form

## Todo:

- [x] Make the HubSpot registration form configurable on an event-by-event basis. Currently hard-coded for _all_ webinars & AMA sessions.
- [x] Hide the Registration form for past Webinars & AMA sessions
- [ ] Replace placeholder events with real events
    - [ ] Webinars
    - [x] Ask Me Anything 

## Demonstration:

https://user-images.githubusercontent.com/99246719/214820770-c18b2eb0-4dbf-48b5-8461-a40ed625d921.mov

## Related Issue(s)

Closes #326 & #358

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [-] Suitable unit/system level tests have been added and they pass
 - [-] Documentation has been updated
    - [-] Upgrade instructions
    - [-] Configuration details
    - [-] Concepts
 - [-] Changes `flowforge.yml`?
    - [-] Issue/PR raised on `flowforge/helm` to update ConfigMap Template
    - [-] Issue/PR raised on `flowforge/CloudProject` to update values for Staging/Production

## Labels

 - [-] Backport needed? -> add the `backport` label
 - [-] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by joepavitt** [comment]
@iskerrett for me to be able to complete this - could you please send me over the `.md` file for the first webinar. If we don't know when that will be yet, I can push just the AMA session for now.

**Comment by iskerrett** [comment]
@robmarcer will be working on the webinar content. 

Please don't push the AMA session yet. I still need to do work on the back end with Zoom Webinar and Hubspot.

**Comment by iskerrett** [comment]
I noticed in the new demonstration that the form is now below the presenters. We really need to have the form on top so people easily see it without scrolling.

**Comment by joepavitt** [comment]
>  I noticed in the new demonstration that the form is now below the presenters. We really need to have the form on top so people easily see it without scrolling.

No problem, will change it back

**Comment by joepavitt** [comment]
I think it does look a little strange this way round, but will stick with it if you'd prefer. The registration form is within view on first load of the page, even if "hosted by" appears first.

Screenshot 2023-01-27 at 12 58 43

**Comment by joepavitt** [comment]
Will go live without webinars for now and will look like this:

Screenshot 2023-01-31 at 15 28 20

File: src/contact-us.njk [hunk 1]
```diff
@@ -2,6 +2,8 @@
 layout: layouts/nohero.njk
 title: Contact us
 sitemapPriority: 0.8
+meta:
+    description: Get in touch with
```
Comment (knolleary) [inline src/contact-us.njk:6]: Is that intentionally cut off mid-sentence so if gets joined with 'FlowForge' in the title?

Comment (joepavitt) [reply src/contact-us.njk:6]: nope, good spot, that's an error

**Review by knolleary — APPROVED** [review]

**Pull request merged**